libs/webutil.js

```
@@ -93,31 +94,40 @@ exports.WebUtil = class WebUtil {
  */
   static async verifyCSS(locator, cssProps) {
     this.locator = locator;
-
     // Verify the CSS properties and values
+    let result = true;
     await Promise.allSettled(
       Object.entries(cssProps).map(async ([property, expectedValue]) => {
-        await expect(await this.locator).toHaveCSS(property, expectedValue).toBeTruthy();
+        try {
```

Do you test this logic, is this working for all cases? because this will catch all exceptions